### DISCUSSION TITLE
Concertation 2.1.0-ballot & mesures de santé

### DISCUSSION THREAD

**Discussion by nriss** [Interop-Sante/hl7.fhir.fr.core#173] [General]
Bonjour à tous!

# Concertation IG FrCore 2.1.0

Suite à des retours implémenteurs, voici une release mineure qui propose quelques correctifs permettant à nouveau d'améliorer la qualité de l'IG FrCore!

Liste des modifications: 
* Ajout des slices INS NIR demo et test pour le patient INS et séparation des slices SIREN et SIRET [163](https://github.com/Interop-Sante/hl7.fhir.fr.core/pull/163)
* Correction du discriminant de Organization.type [166](https://github.com/Interop-Sante/hl7.fhir.fr.core/pull/166)
* Ajout des tags MS sur les champs obligatoires dans le cadre de lINS du PatientINS & ajout dinformations [167](https://github.com/Interop-Sante/hl7.fhir.fr.core/pull/167), [169](https://github.com/Interop-Sante/hl7.fhir.fr.core/pull/169)
* Fusion des profils PractitionerRole [61](https://github.com/Interop-Sante/hl7.fhir.fr.core/pull/61) déjà discuté ici : https://github.com/Interop-Sante/hl7.fhir.fr.core/discussions/165

Lien d'accès : https://hl7.fr/ig/fhir/core/2.1.0-ballot

De plus, cela permettra de corriger le bug d'affichage des "XXXXX" un peu partout venant de la version du publisher utilisé au moment de la publication de 2.0.1

<img width="1265" alt="image" src="https://github.com/user-attachments/assets/73b6eabc-21c0-422a-a3ef-a1146f03c3c7">

Release officielle estimée : début septembre

# Mesures de santé

Je profite également de cette concertation pour vous demander un avis : 
Je propose de supprimer les profils mesures de FrCore pour les concentrer dans l'IG [mesures de santé](https://interop.esante.gouv.fr/ig/fhir/mesures)

Il y a certes un héritage, mais cela fait doublon et complexifie l'écosystème Fr 

Objectif : supprimer les mesures de FrCore 2.1.0, mettre à jour l'IG mesures pour enlever les héritages des profils et garder toute la cohérence actuelle

**Comment by davidouagne** [comment]
Bonjour,

Dans le cadre du groupe de travail "standards et interopérabilité" lancé en janvier 2024 et piloté par le HDH, un sous-groupe FHIR a été constitué pour utiliser FHIR, avec pour objectif de produire un guide d'implémentation (IG) comme livrable. Ce groupe vise à standardiser les variables essentielles pour les Entrepôts de données de santé (EDS).

Parmi les variables essentielles, on retrouve le poids, la taille et la pression artérielle, soulevant la question : quelle est la meilleure à inclure (FrCore ou Mesure) ? Il est entendu qu'il n'est pas question de repartir de zéro depuis le standard.

Voici ma conclusion : les StructureDefinition de FrCore sont plus adaptées à mes besoins que celles issues de Mesure. En effet, le groupe de travail s'est concentré sur un usage en santé publique, où une date, un code, une valeur et une unité suffisent. Mesure rend des attributs supplémentaires "Must Supported" qui, dans mon contexte, représentent des contraintes inutiles.

FrCore devrait couvrir toutes les variables essentielles à la manière des États-Unis, comme illustré ici : https://www.healthit.gov/isp/united-states-core-data-interoperability-uscdi#uscdi-v5 de le FIG US Core. Cependant, sauf erreur de ma part, nous ne disposons pas d'une liste équivalente en France. Dans FrCore, les variables doivent être aussi réutilisables que possible (peu ou pas de contraintes sur les cardinalités et une utilisation minimale des MS), seules les adaptations spécifiques au contexte français doivent être mise en oeuvre. Ensuite, chaque variable (StructureDefinition) pourra être spécialisée pour des cas d'usage opérationnels, comme le fait Mesure.
